layersvt/device_simulation.cpp

```diff
@@ -4246,6 +4246,59 @@ VKAPI_ATTR void VKAPI_CALL GetPhysicalDeviceFormatProperties2KHR(VkPhysicalDevic
     GetPhysicalDeviceFormatProperties2(physicalDevice, format, pFormatProperties);
 }
 
+VKAPI_ATTR VkResult VKAPI_CALL GetPhysicalDeviceImageFormatProperties(VkPhysicalDevice physicalDevice, VkFormat format,
+                                                                      VkImageType type, VkImageTiling tiling,
+                                                                      VkImageUsageFlags usage, VkImageCreateFlags flags,
+                                                                      VkImageFormatProperties *pImageFormatProperties) {
+    std::lock_guard<std::mutex> lock(global_lock);
```

Comment:
When running the test in issue #1604, it got stuck [when trying to get the lock in `GetPhysicalDeviceFormatProperties()`](https://github.com/LunarG/VulkanTools/pull/1607/files#diff-a900e335f1854d7e8388801e46a065687ff1686fc8f97f77437656eb309b98c8R4155), because the lock was already grabbed in this line.

Reply:
Should be fixed now.